src/contrib/profiling-sssd.rst

```diff
@@ -0,0 +1,116 @@
+Profiling SSSD
+##############
+
+Getting Ready
+*************
+
+There are several tools allowing us to profile an application. We will focus on
+perf_ which seems to be the one that works best with SSSD.
+
+.. _perf: https://perf.wiki.kernel.org
+
+First you need to install the ``perf`` tool:
+
+.. code-tabs::
+
+    .. fedora-tab::
+
+        # dnf -y install perf js-d3-flame-graph
+
+    .. ubuntu-tab::
+
+        $ sudo apt install linux-tools-common linux-tools-generic
+
+
+Make sure the debug information is available for SSSD and all its dependencies
+either by using ``debuginfod``, or by manually installing the debug information
+packages:
+
+.. code-tabs::
+
+    .. fedora-tab::
+
+        # dnf -y --enable-repo=fedora-debuginfo --enable-repo=updates-debuginfo \
+        install sssd*debuginfo libsss_*-debuginfo samba-client-libs-debuginfo \
+        libldb-debuginfo libtevent-debuginfo libtalloc-debuginfo glibc-debuginfo
```

Comment:
Maybe `dnf debuginfo-install sssd`?

Reply:
Or will it miss debug info for deps..?

Reply:
Also `libcares`, `libtdb`, `libdhash`, `libldap`...

Reply:
I didn't know the first command. I think it is better to use it instead of repeating `-debuginfo` on every package. Nevertheless, it also install the `-debugsource` packages which, I think, are not required. I will check that.

In addition, it doesn't resolve the dependencies, so every single package must still be mentioned explicitly.

Debuginfo for `libtdb` is included in `libldb-debuginfo`. Iḿ adding the other you mentioned and a few more.

Reply:
After checking, `debuginfo-install` does resolve the dependencies for other packages. So I guess there might be some problem with ours.

Reply:
> sssd* libsss_*

Doesn't simple "sssd" work (installing everything, at least for SSSD)?

Reply:
Looks like "no".